tools/vms-generator/utils/utils.go

```diff
@@ -84,6 +84,7 @@ const (
 )
 
 const windowsFirmware = "5d307ca9-b3ef-428c-8861-06e72d69f223"
+const defaultName = "default"
```

Comment:
default name of what? please rename into `defaultInterfaceName` or `defaultNetworkName`.